sql-prepared-plan-cache.md

```diff
@@ -190,12 +192,12 @@ Using Prepared Plan Cache has some memory overhead. In internal tests, each cach
 
 For example, the current TiDB instance has 50 sessions in concurrency and each session has approximately 100 cached plans. The total memory consumption is approximately `50 * 100 * 100 KiB` = `512 MB`.
```

Comment:
@qw4990 Please confirm whether the following TiDB Cloud content needs to be modified
```
<CustomContent platform="tidb-cloud">
Using Prepared Plan Cache has some memory overhead. In internal tests, each cached plan consumes an average of 100 KiB of memory. Because Plan Cache is currently at the `SESSION` level, the total memory consumption is approximately `the number of sessions * the average number of cached plans in a session * 100 KiB`.

For example, the current TiDB instance has 50 sessions in concurrency and each session has approximately 100 cached plans. The total memory consumption is approximately `50 * 100 * 100 KiB` = `512 MB`.
```